src/components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import '../App.css';
+
+function Calculator() {
+  return (
+    <div className="container flex">
+      <div className="child result flex">
+        <p>0</p>
+      </div>
+      <div className="child flex">
+        <button type="button" className="button key">AC</button>
+        <button type="button" className="button key">+/-</button>
+        <button type="button" className="button key">%</button>
+        <button type="button" className="button operator">+</button>
```

Comment:
- [ ] I will advise you to use the divide symbol `÷`. You must have made a mistake here. This is for ordinary users who are not into the technical aspect of coding to understand it.
### Your design
![image](https://user-images.githubusercontent.com/86200356/220566447-a50cdc92-78fa-45f0-8c68-4578cf098764.png)

### Wireframe
![image](https://user-images.githubusercontent.com/86200356/220567042-2f5a16d9-3809-4b76-a2ac-07cd1bce0d13.png)